modules/virt-creating-template-redhat-web.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-creating-template-redhat-web_{context}"]
+= Creating a virtual machine template from a Red Hat template in the web console
+
+Select a virtual machine template provided by Red Hat and customize the template by adding a boot source and saving the template. The customized virtual machine template is available for you to create multiple virtual machines.
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
+. Click the *Templates* tab.
+. Identify the Red Hat template that you want to use for creating a virtual machine template. If the Red Hat template was not configured with a boot source, click *Add source* and continue with the required steps in the *Add boot source to template* dialog. If the Red Hat template was configured with a boot source and displays *Available* for boot source, click the Options menu {kebab}, select *Create new template from* and complete the steps in the interactive wizard.
+. In the *Add boot source to template* dialog, click *Boot source type*. Select a method for creating a persistent volume claim (PVC): *Import local file*, *Import via URL*, *Clone existing PVC*, or *Import via Registry*.
+. Enter a value for *Persistent Volume Claim size* to specify the PVC size that is adequate for the uncompressed image and any additional space that is required.
+
++
+[NOTE]
```

Comment:
This note is not necessary as the doc is not about Virtual Machine.

Reply:
Removed the note.